app-conf/GeneralConf.xml

```diff
@@ -37,4 +37,20 @@
     <value>true</value>
     <description>If this property is "false", search will only make exact matches</description>
   </property>
+  <!--
+  Flag to enable to fetch only recently finished apps, by default it fetches apps finished in last one hour.
+  -->
+  <!--
+  <property>
+    <name>drelephant.analysis.only.fetchRecentApps</name>
+    <value>true</value>
+  </property> -->
+  <!--
+  Property to change to how much older apps to fetch from RM. The default value is 3600000 (one hour)
+  -->
+  <!--
+  <property>
+    <name>drelephant.analysis.fetch.startTimeWindowMillis</name>
```

Comment:
Do we really require 2 configs? I would suggest having a single config called ```drelephant.analysis.fetch.initial.window``` set to 3600000